src/main/java/edu/greenblitz/robotName/subsystems/Arm/Wrist.java

```diff
@@ -0,0 +1,104 @@
+package edu.greenblitz.robotName.subsystems.Arm;
+
+
+import com.revrobotics.CANSparkMax;
+import edu.greenblitz.robotName.subsystems.Arm.EndEffector.WristUtils.IWrist;
+import edu.greenblitz.robotName.subsystems.Arm.EndEffector.WristUtils.NeoWrist.NeoWristConstants;
+import edu.greenblitz.robotName.subsystems.Arm.EndEffector.WristUtils.WristFactory;
+import edu.greenblitz.robotName.subsystems.Arm.EndEffector.WristUtils.WristInputsAutoLogged;
+import edu.greenblitz.robotName.subsystems.Battery;
+import edu.greenblitz.robotName.utils.GBSubsystem;
+import org.littletonrobotics.junction.Logger;
+
+import static edu.greenblitz.robotName.subsystems.Arm.EndEffector.WristUtils.WristConstants.TOLERANCE;
+
+public class Wrist extends GBSubsystem {
+
+    private static Wrist instance;
+
+    private WristInputsAutoLogged wristInputs;
+
+    private IWrist wrist;
```

Comment:
Again, kinda weird that a class called `Wrist` holds an instance of itself and and `IWrist wrist`

Reply:
same thing about elbow

Reply:
> Again, kinda weird that a class called `Wrist` holds an instance of itself and and `IWrist wrist`

It's convention.